salesforce_api/gsheet-new-row-to-sfdc-new-contact/sample.bal

```diff
@@ -0,0 +1,52 @@
+import ballerina/log;
```

Comment:
Why having the file name as sample.bal? Instead keep the default of `main.bal`

Reply:
This came with the style used for the choreo samples in wso2-enterprice. Updated in [bcf09ac](https://github.com/ballerina-guides/integration-samples/pull/31/commits/bcf09ac42e87543485854586312facacacd050cb)